meson.build

```diff
@@ -49,6 +49,12 @@ endif
 
 assert(pybind11.found(), 'Could not find pybind11!')
 
+if get_option('problem') == 'cvrp'
+    # CVRP does not have time windows, so we set a flag that compiles time 
+    # window stuff out of the extension modules.
+    add_project_arguments('-DVRP_NO_TIME_WINDOWS', language: 'cpp')
+endif
+
```

Comment:
Yes, I'll have a look at this. My first look will be at "why didn't gcc give me this overview" :).

Reply:
This is a clang-specific warning, so gcc doesn't have it (yet). I'm disabling it when compiling for CVRP, since those unused fields are all TW-related and it's OK they're still there.